src/multi_physics/QED/src/physics/quantum_sync/quantum_sync_engine_tabulated_functions.hpp

```diff
@@ -0,0 +1,294 @@
+#ifndef PICSAR_MULTIPHYSICS_QUANTUM_SYNC_ENGINE_TABULATED_FUNCTIONS
+#define PICSAR_MULTIPHYSICS_QUANTUM_SYNC_ENGINE_TABULATED_FUNCTIONS
+
+//This .hpp file contais the implementation of the functions
+//tabulated in the lookup tables for the Quantum Synchrotron photon emission engine.
+//Please have a look at the jupyter notebook "validation.ipynb"
+//in QED_tests/validation for a more in-depth discussion.
+//
+// References:
+// 1) C.P.Ridgers et al. Journal of Computational Physics 260, 1 (2014)
+// 2) A.Gonoskov et al. Phys. Rev. E 92, 023305 (2015)
+
+//Should be included by all the src files of the library
+#include "../../qed_commons.h"
+
+//Uses physical constants
+#include "../phys_constants.h"
+//Uses mathematical constants
+#include "../../math/math_constants.h"
+//Uses integration routines
+#include "../../math/quadrature.hpp"
+//Uses special functions
+#include "../../math/special_functions.hpp"
+//Uses sqrt and cbrt
+#include "../../math/cmath_overloads.hpp"
+
+#include <algorithm>
+#include <stdexcept>
+
+namespace picsar{
+namespace multi_physics{
+namespace phys{
+namespace quantum_sync{
+
+    /**
+    * It computes the y parameter (see validation script).
+    * This function is not designed to be run on GPUs.
+    *
+    * @tparam RealType the floating point type to be used
+    *
+    * @param[in] chi_phot the chi parameter of the photon
+    * @param[in] chi_ele the chi parameter of the electron
+    *
+    * @return the parameter X
```

Comment:
This does not seem to correspond to this function

Reply:
yes, you are right!